tests/resources/VCH-Util.robot

```diff
@@ -88,22 +88,23 @@ Set Test Environment Variables
     Run Keyword If  ${status}  Wait Until Keyword Succeeds  5x  1s  Create Unique Bridge Network
 
 Create Unique Bridge Network
+    #Retry 3 times if ${rc} not return 0; there might be vlan conflict while parallel tests are running.
+    :FOR  ${index}  IN RANGE  3
     # Ensure unique bridges are non-overlapping in a shared build environment (our CI)
-    @{URLs}=  Split String  %{TEST_URL_ARRAY}
-    ${idx}=  Get Index From List  ${URLs}  %{TEST_URL}
-    ${lowerVLAN}=  Evaluate  (${idx}+2) * 100
-    ${upperVLAN}=  Evaluate  ${lowerVLAN}+100
+    \  @{URLs}=  Split String  %{TEST_URL_ARRAY}
+    \  ${idx}=  Get Index From List  ${URLs}  %{TEST_URL}
+    \  ${lowerVLAN}=  Evaluate  (${idx}+2) * 100
+    \  ${upperVLAN}=  Evaluate  ${lowerVLAN}+100
 
     # Set a unique bridge network for each VCH that has a random VLAN ID
-    ${vlan}=  Evaluate  str(random.randint(${lowerVLAN}, ${upperVLAN}))  modules=random
-    ${vswitch}=  Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Run  govc host.vswitch.info -json | jq -r ".Vswitch[0].Name"
-    ${rc}  ${output}=  Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Run And Return Rc And Output  govc host.portgroup.add -vlan=${vlan} -vswitch ${vswitch} VCH-%{DRONE_BUILD_NUMBER}-${vlan}
-    Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Should Be Equal As Integers  ${rc}  0
-
-    ${dvs}=  Run Keyword If  '%{HOST_TYPE}' == 'VC'  Run  govc find -type DistributedVirtualSwitch | head -n1
-    ${rc}  ${output}=  Run Keyword If  '%{HOST_TYPE}' == 'VC'  Run And Return Rc And Output  govc dvs.portgroup.add -vlan=${vlan} -dvs ${dvs} VCH-%{DRONE_BUILD_NUMBER}-${vlan}
-    Run Keyword If  '%{HOST_TYPE}' == 'VC'  Should Be Equal As Integers  ${rc}  0
-    Set Environment Variable  BRIDGE_NETWORK  VCH-%{DRONE_BUILD_NUMBER}-${vlan}
+    \  ${vlan}=  Evaluate  str(random.randint(${lowerVLAN}, ${upperVLAN}))  modules=random
+    \  ${vswitch}=  Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Run  govc host.vswitch.info -json | jq -r ".Vswitch[0].Name"
+    \  ${rc}  ${output}=  Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Run And Return Rc And Output  govc host.portgroup.add -vlan=${vlan} -vswitch ${vswitch} VCH-%{DRONE_BUILD_NUMBER}-${vlan}
+
+    \  ${dvs}=  Run Keyword If  '%{HOST_TYPE}' == 'VC'  Run  govc find -type DistributedVirtualSwitch | head -n1
+    \  ${rc}  ${output}=  Run Keyword If  '%{HOST_TYPE}' == 'VC'  Run And Return Rc And Output  govc dvs.portgroup.add -vlan=${vlan} -dvs ${dvs} VCH-%{DRONE_BUILD_NUMBER}-${vlan}
+    \  Run Keyword If  ${rc} == 0  Run Keyword And Return  Set Environment Variable  BRIDGE_NETWORK  VCH-%{DRONE_BUILD_NUMBER}-${vlan}
+    \  ...  ELSE  Log  ${output}
```

Comment:
It may be useful to log this at WARN level as suggested in #8061 to make it easier to see when we've encountered a retry.

> This is unlikely to be an easy failure to recreate unless artificially bounding the number of available VLANs tightly. We have two testing options:
> 1. determine if when we hit the scenario and that we cleanly try a different VLAN
> 2. run enough builds with "high enough" parallelism that our confidence is good that it's addressed.
> 
> I'm strongly biased towards positive affirmation of fixes so will request (1). To make this easier than grepping through output.xml I suggest adding a WARN statement (see [here](https://github.com/vmware/vic/pull/6943/files#diff-29742001d5fccddbf03a5092aad4b847R395)), using that to determine when a PR build hits the issue, and then confirming in output.xml that we behaved as expected.

Reply:
I've updated what test I have done to prove the re-try logic works as expected.

Reply:
From below logs from https://ci-vic.vmware.com/vmware/vic/19652, we can observe the re-try works, and allocated a new vlan when the conflict happens.
![screen shot 2018-08-01 at 5 05 02 pm](https://user-images.githubusercontent.com/14900967/43512234-1b10ad54-95ad-11e8-82d6-4bdb44b9b591.png)